pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -55,16 +56,22 @@ class VmOverviewTabLibvirt extends React.Component {
     }
 
     render() {
+        const memModalAction = (providerName) => {
```

Comment:
If the provider is virsh this will return null. And it will show no memory in the UI.